index.js

```diff
@@ -99,6 +99,14 @@ mOcKiNgCaSe.overrideString = () => {
   return mOcKiNgCaSe;
 };
 
+mOcKiNgCaSe.overrideConsole = (options = {}) => {
+  const print = { ...console };
```

Comment:
Is destructuring necessary here??   
`const originalConsole = console;` should work